extensions/trace_utils/src/main/java/io/opentelemetry/extensions/trace/CurrentSpanUtils.java

```diff
@@ -121,7 +121,7 @@ public V call() throws Exception {
 
   private static void setErrorStatus(Span span, Throwable t) {
     span.setStatus(
-        Status.UNKNOWN.withDescription(
+        Status.ERROR.withDescription(
```

Comment:
I realised that I still an open question about this use case: https://github.com/open-telemetry/opentelemetry-specification/issues/1008

Reply:
I'd be happy to revisit this when the issue is clarified. Meanwhile, it does seem like approximately preserving the existing behavior is a good thing for now. And, as I commented on that issue, I think the intent with `recordException` is that it's an error, most commonly.

Reply:
When that gets merged, I'll create an issue to address it here. Meanwhile, keeping the existing behavior seems like the right decision for this PR.